tools/dgeni/processors/categorizer.js

```diff
@@ -171,6 +171,29 @@ function getDirectiveOutputAlias(doc) {
   return isDirectiveOutput(doc) ? doc.decorators.find(d => d.name == 'Output').arguments[0] : '';
 }
 
+function getDirectoveSelectors(classDoc) {
+  let blacklist = [
+    '[portal]',
+    '[portalHost]',
+    'textarea[md-autosize]',
+    '[overlay-origin]',
+    '[connected-overlay]',
+  ];
```

Comment:
Good idea. I had thought of that, but decided to go with this since it seemed specific to this function. I'll make these changes once I get the CLA figured out.